packages/compass-e2e-tests/helpers/commands/add-database.ts

```diff
@@ -1,10 +1,17 @@
 import type { CompassBrowser } from '../compass-browser';
 import * as Selectors from '../selectors';
 
+type AddDatabaseOptions = {
+  // Intentionally empty. Just here as placeholder to make it the same as addCollection
```

Comment:
what is the advantage of this?

Reply:
I have to call the method with parameters after options.

Reply:
i'm sorry .. i still don't get it, what's wrong with:
 
```
  await browser.addDatabase(
      dbName,
      collectionName,
      'add-database-modal-basic.png'
    );
 ```
 
 btw super non-important, i'm just curious